cmd/admin-handlers.go

```diff
@@ -142,41 +173,60 @@ func (a adminAPIHandlers) ServerUpdateHandler(w http.ResponseWriter, r *http.Req
 		return
 	}
 
-	for _, nerr := range globalNotificationSys.DownloadBinary(ctx, u, sha256Sum, releaseInfo) {
-		if nerr.Err != nil {
-			err := AdminError{
-				Code:       AdminUpdateApplyFailure,
-				Message:    nerr.Err.Error(),
-				StatusCode: http.StatusInternalServerError,
+	// Get client's architecture to send proper file
```

Comment:
I don't completly understand this part, we should send the binary file based on the server architecture instead of client right? otherwise if my client is amd and my server is arm im going to end with an amd binary on the server? what would happen if I have a minio cluster in where all nodes have different architectures?

Reply:
I already told this in above review @Alevsk